src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java

```diff
@@ -0,0 +1,319 @@
+		//Note: only MSSQL mappings here.
+	}
+
+	/**
+	 * MsSQL specific time based injection strings.
+	 * Note: <<<<ORIGINALVALUE>>>> is replaced with the original parameter value at runtime in these examples below (see * comment)
+	 * TODO: Add SQL_MSSQL_TIME_REPLACEMENTS queries related to PARAM in SELECT/UPDATE/DELETE clause
+	*/
+	private static String[] SQL_MSSQL_TIME_REPLACEMENTS = {
+		ORIG_VALUE_TOKEN + " WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,						//Param in WHERE clause
+		ORIG_VALUE_TOKEN + "' WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,						//Param in WHERE clause
+		ORIG_VALUE_TOKEN + "\" WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,						//Param in WHERE clause
+		ORIG_VALUE_TOKEN + ") WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ") ' WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ") \" WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ")) WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ")) ' WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ")) \" WAITFOR DELAY '" + SLEEP_TOKEN + "'" + SQL_ONE_LINE_COMMENT,
+		ORIG_VALUE_TOKEN + ") WAITFOR DELAY '" + SLEEP_TOKEN + "' (" ,
+		ORIG_VALUE_TOKEN + ") ' WAITFOR DELAY '" + SLEEP_TOKEN + "' (",
+		ORIG_VALUE_TOKEN + ") \" WAITFOR DELAY '" + SLEEP_TOKEN + "' (",
+		ORIG_VALUE_TOKEN + ")) WAITFOR DELAY '" + SLEEP_TOKEN + "' ((",
+		ORIG_VALUE_TOKEN + ")) ' WAITFOR DELAY '" + SLEEP_TOKEN + "' ((",
+		ORIG_VALUE_TOKEN + ")) \" WAITFOR DELAY '" + SLEEP_TOKEN + "' ((",
+	};
+
+	/**
+	 * plugin dependencies (none! not even "SQL Injection")
+	 */
+	private static final String[] dependency = {};
+
+	/**
+	 * for logging.
+	 */
+	private static Logger log = Logger.getLogger(SQLInjectionMsSQL.class);
+
+	/**
+	 * determines if we should output Debug level logging
+	 */
+	private boolean debugEnabled = log.isDebugEnabled();
+
+	@Override
+	public int getId() {
+		return 40027;
+	}
+
+	@Override
+	public String getName() {
+		return Constant.messages.getString("ascanalpha.sqlinjection.mssql.name");
+	}
+
+	@Override
+	public String[] getDependency() {
+		return dependency;
+	}
+
+	@Override
+	public boolean targets(TechSet technologies) {
+		return technologies.includes(Tech.MsSQL);
+	}
+
+	@Override
+	public String getDescription() {
+		return Constant.messages.getString("ascanalpha.sqlinjection.mssql.desc");
+	}
+
+	@Override
+	public int getCategory() {
+		return Category.INJECTION;
+	}
+
+	@Override
+	public String getSolution() {
+		return Constant.messages.getString("ascanalpha.sqlinjection.mssql.soln");
+	}
+
+	@Override
+	public String getReference() {
+		return Constant.messages.getString("ascanalpha.sqlinjection.mssql.refs");
+	}
+
+	@Override
+	public void init() {
+		if ( this.debugEnabled ) log.debug("Initialising");
```

Comment:
Should have the braces and the log statement be in its own line (same for all other if statements).